index.js

```diff
@@ -199,6 +199,16 @@ exports.sendMoney = async function (options) {
     await eventtracking.track(eventtracking.EVENT_ID_SEND_TOKENS_END, { node: options.nodeUrl, success: true });
 };
 
+exports.deleteAccessKey = async function (options) {
```

Comment:
nit: I'd prefer we add new commands as separate self-contained files (i.e. in `commands/` folder). We should extract existing commands too eventually.

Reply:
okay, pushed up that change, Vlad